examples/import-key/versions.tf

```diff
@@ -0,0 +1,11 @@
+#########################################################################################
```

Comment:
Can we upgrade this versions.tf to support terraform version 13

Reply:
I believe this supports version >0.13

Reply:
Including parameter following parameter enforce user to have terraform v13 (even aws, and google follows the same)
"required_version = ">= 0.13"